lib/galaxy/tools/imp_exp/export_history.py

```diff
@@ -100,7 +100,15 @@ def main():
     parser.add_option('-G', '--gzip', dest='gzip', action="store_true", help='Compress archive using gzip.')
     (options, args) = parser.parse_args()
     gzip = bool(options.gzip)
-    history_attrs, dataset_attrs, job_attrs, out_file = args
+    if len(args) == 2:
```

Comment:
Could we future proof it by adding a `galaxy_version` flag to the argument parser ?

Reply:
I'll work on this - but this seems like overkill we don't apply to other scripts. What if we just used named arguments for everything instead?

Reply:
Sure, that'd be fine.